src/store/RelayRecordWriter.js

```diff
@@ -34,6 +34,7 @@ import type {
 import type {RecordState} from 'RelayRecordState';
 const RelayRecordStatusMap = require('RelayRecordStatusMap');
 import type {CacheWriter} from 'RelayTypes';
+const stableStringify= require('stableStringify');
```

Comment:
nit: we follow a convention that function modules are grouped separately than class/object imports - this should go after the block around lines 39-40